test/parallel_api/algorithm/alg.sorting/alg.min.max/max_comp.pass.cpp

```diff
@@ -0,0 +1,60 @@
+//===----------------------------------------------------------------------===//
```

Comment:
It looks that we have wrong file "header" here.

Reply:
Fixed